bin/apisix

```diff
@@ -252,6 +252,7 @@ http {
 
         ssl_protocols {* ssl.ssl_protocols *};
         ssl_ciphers {* ssl.ssl_ciphers *};
+        ssl_prefer_server_ciphers on;
```

Comment:
do we need to add a new switch to control this flag?

Reply:
other PR will fix this.